src/model/Global.ts

```diff
@@ -238,6 +252,9 @@ class GlobalModel extends Model<ECUnitOption> {
 
         if (!type || type === 'recreate') {
             const baseOption = optionManager.mountOption(type === 'recreate');
+            if (__DEV__) {
+                checkMissingComponents(baseOption);
```

Comment:
Seems the code of `checkMissingComponents` will be bundled into dist files. I remember this is only used in development. Is that expected?

Reply:
Oh. I missed it. It won't.